app/src/main/java/server/RouteObject.java

```diff
@@ -0,0 +1,24 @@
+package server;
+
+import java.util.*;
+
+interface RouteInterface {
+    String getResponse(String HTTPVersion, String headers, String body);
+}
+
+public class RouteObject {
+    String route;
+    ArrayList<String> methods;
+    RouteInterface objectResponse;
+
+    public RouteObject(String objectRoute, ArrayList<String> objectHeaders, RouteInterface response){
+        this.route = objectRoute;
+        this.methods = objectHeaders;
```

Comment:
do you mean `this.methods = objectMethods` ?

Reply:
changed to objectMethods

Reply:
Changed again in the naming scheme suggested below